README.md

```diff
@@ -25,14 +25,14 @@ We are moving development of Fractal v2.0 and all first-party add-ons into a 'mo
 #### Running locally
 
 1. Clone this repository
-2. Install dependencies - `npm install`
-3. Bootstrap the packages together using Lerna - `npm run bootstrap`
+2. Install dependencies - `yarn install`
```

Comment:
So..
1) yes to swapping out for npm in scripts
2) I hadn't considered that... I _think_ you can use the `script` property in the travis.yml to specify what npm script to use so could create a specific CI one: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Using-Gulp